test/unit/PostSettlementSurplusAuctionHouse.t.sol

```diff
@@ -490,16 +504,30 @@ contract Unit_PostSettlementSurplusAuctionHouse_IncreaseBidSize is Base {
     uint256 _bidIncrease,
     uint256 _bidDuration
   ) public happyPath(_auction, _bid, _bidIncrease, _bidDuration) {
+    uint256 _payment = _bid;
+    uint256 _refund;
+
+    // if the user was not the previous high bidder we refund the previous high bidder them
+    if (_auction.highBidder != user && _auction.bidExpiry != 0) {
+      _refund = _auction.bidAmount;
+      _payment = _bid - _auction.bidAmount;
+
+      // if the user was the previous high bidder they only need to pay the increment
+    } else if (_auction.highBidder == user && _auction.bidExpiry != 0) {
+      _payment = _bid - _auction.bidAmount;
+    }
+
+    if (_refund != 0) {
+      vm.expectCall(
+        address(mockProtocolToken),
+        abi.encodeCall(mockProtocolToken.transferFrom, (user, _auction.highBidder, _refund)),
+        1
```

Comment:
In this test, on the contrary, the `_refund` should always be a positive number, unless `_auction.bidExpiry == 0`.